package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mapboxgl-powerbi",
-  "version": "20201201",
+  "version": "1.5.0",
```

Comment:
this is actually describing the node version needed to compile the code, reverted to 1.5.0